articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md

```diff
@@ -68,3 +62,5 @@ If Auth0 acts as the identity provider, you need to make the following changes t
   https://${account.namespace}/samlp
   /${account.clientId}?RelayState=http://FINAL_DESTINATION_URL
```

Comment:
Can you change the RelayState to `https://`

Reply:
done